themes/bootstrap3/templates/RecordDriver/SolrDefault/result-list.phtml

```diff
@@ -164,6 +164,10 @@
           </span>
         <? endif; ?>
 
+        <? if ($this->cart()->isActiveInSearch()): ?>
```

Comment:
The equivalent line in EDS/result-list.phtml above is `<? if ($this->cart()->isActiveInSearch() && (!isset($this->showCartControls) || $this->showCartControls)): ?>` -- are these intentionally out of sync, or did one file get updated without the other?